snuba/cli/replacer.py

```diff
@@ -27,7 +27,7 @@
     "--dataset",
     "dataset_name",
     default="events",
-    type=click.Choice(["events"]),
+    type=click.Choice(["events", "events_migration"]),
```

Comment:
I still find this confusing

Reply:
The names or listing the datasets instead of taking them from factory ?

Reply:
The names

Reply:
The problem is that we need to swap the errors dataset with the events dataset during the rollout (and call it events) since sentry references it. Thus we cannot call it errors. After the migration we can rename everything.
We could have two datasets during the migration, once called errors and one that will replace events and then change sentry, but having two datasets around seems at least as confusing as the "events_migration" name.